package/imx-mkimage/imx-mkimage.mk

```diff
@@ -4,26 +4,25 @@
 #
 ################################################################################
 
-IMX_MKIMAGE_VERSION = lf-5.10.72_2.2.0
+IMX_MKIMAGE_VERSION = lf-6.1.22-2.0.0
```

Comment:
check to see if lf-6.1.36-2.1.0 is available?

Reply:
updated to lf-6.1.36-2.1.0